olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.cpp

```diff
@@ -517,8 +517,18 @@ bool VersionedLayerClientImpl::RemoveFromCache(
 }
 
 bool VersionedLayerClientImpl::RemoveFromCache(const geo::TileKey& tile) {
-  auto partition_id = tile.ToHereTile();
-  return RemoveFromCache(partition_id);
+  read::QuadTreeIndex cached_tree;
+  if (repository::PartitionsRepository::FindQuadTree(
+          catalog_, settings_, layer_id_, catalog_version_.load(), tile,
+          cached_tree)) {
+    auto data = cached_tree.Find(tile, false);
+    if (data) {
+      repository::DataCacheRepository cache_repository(catalog_,
+                                                       settings_.cache);
+      return cache_repository.Clear(layer_id_, data->data_handle);
```

Comment:
Please keep consistent. Above we have data_repository, you use cache_repository. Please use same as above.